BioForm/BioForm/settings.py

```diff
@@ -144,3 +146,7 @@
 LOGIN_EXEMPT_URLS = (
     r'^profiles/register/$',
 )
+
+CORS_ORIGIN_ALLOW_ALL = True
+CORS_ALLOW_CREDENTIALS = True
```

Comment:
As you have removed CORS middleware so these configs are not required anymore.

Reply:
Changed as per suggestion